core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -506,7 +506,8 @@ UNWIND(	.cantunwind)
 
 #if defined (CFG_BOOT_SECONDARY_REQUEST)
 	/* if L1 is not invalidated before, do it here */
-	bl	arm_cl1_d_invbysetway
+	mov	r0, #DCACHE_OP_INV
+	bl	dcache_op_all
```

Comment:
```
%grep -l CFG_BOOT_SECONDARY_REQUEST **/*.mk
core/arch/arm/plat-imx/conf.mk
core/arch/arm/plat-ls/conf.mk
core/arch/arm/plat-zynq7k/conf.mk
mk/config.mk
```
So any of `plat-imx`, `plat-ls` or `plat-zynq7k` should be useful.
Any volunteers/comments, @yanyan-wrs , @sorenb-xlnx , @b49020 ?